bindings/python/py_src/safetensors/torch.py

```diff
@@ -33,8 +34,19 @@ def save_file(
 def load_file(filename: str) -> Dict[str, torch.Tensor]:
     result = {}
     with safe_open(filename, framework="pt") as f:
-        for k in f.keys():
-            result[k] = f.get_tensor(k)
+        with open(filename, mode="r", encoding="utf8") as file_obj:
+            with mmap.mmap(file_obj.fileno(), length=0, access=mmap.ACCESS_READ) as mmap_obj:
```

Comment:
had a question/problem here:

When:
* `access=mmap.ACCESS_READ`, torch warns: UserWarning: The given buffer is not writable, and PyTorch does not support non-writable tensors. This means you can write to the underlying (supposedly non-writable) buffer using the tensor. You may want to copy the buffer to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program.
* `access=mmap.ACCESS_READ`, it errs with `Permission Denied`

Reply:
closed by https://github.com/huggingface/safetensors/pull/32/commits/323f2cc845a38972f1e4a70b19be638aa9b3e1f7